src/app/Notifications/VerifyEmail.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Auth\Notifications\VerifyEmail as VerifyEmailNotification;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class VerifyEmail extends VerifyEmailNotification
+{
+    protected function buildMailMessage($url)
```

Comment:
1.) Add docbloc without parameters and return types.
2.) Add return & parameter types.